seaworthy/containers/provided.py

```diff
@@ -167,6 +167,19 @@ def list_vhosts(self):
         """
         return output_lines(self.exec_rabbitmqctl('list_vhosts'))
 
+    def list_queues(self, vhost):
```

Comment:
Could we do `vhost=None` and then default to `self.vhost`? I see `list_policies` doesn't even let you specify a vhost :/

Reply:
Good point.